src/mpv/events.rs

```diff
@@ -71,31 +71,35 @@ impl Mpv {
     }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Clone, Debug)]
 /// Data that is returned by both `GetPropertyReply` and `PropertyChange` events.
 pub enum PropertyData<'a> {
     Str(&'a str),
     OsdStr(&'a str),
     Flag(bool),
     Int64(i64),
     Double(ctype::c_double),
+    Node(&'a MpvNode),
 }
 
 impl<'a> PropertyData<'a> {
-    fn from_raw(format: MpvFormat, ptr: *mut ctype::c_void) -> Result<PropertyData<'a>> {
+    // SAFETY: meant to extract the data from an event property. See `mpv_event_property` in
+    // `client.h`
+    unsafe fn from_raw(format: MpvFormat, ptr: *mut ctype::c_void) -> Result<PropertyData<'a>> {
```

Comment:
I have made this entire `fn` unsafe. If someone passes in the wrong pointer or a mismatched format, it's UB.